src/sessions.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable github/no-then */
```

Comment:
This will be removed in https://github.com/mongodb/node-mongodb-native/pull/4053